http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/leoPubsubMessageSubscriberModels.scala

```diff
@@ -731,7 +749,7 @@ sealed trait PubsubHandleMessageError extends NoStackTrace {
 }
 object PubsubHandleMessageError {
   final case class PubsubKubernetesError(dbError: AppError,
-                                         appId: AppId,
+                                         appId: Option[AppId],
```

Comment:
There were two `PubsubKubernetesError` `case class`es, one mostly duplicating the other. The above change allowed removing one.